docker/README.md

```diff
@@ -19,3 +19,40 @@ Build the container, for python users:
 After build finished, run the container:
 
     docker run --rm -it lightgbm
+
+## Using CLI via Docker
```

Comment:
Using CLI Version of LightGBM via Docker

Reply:
I'm not convinced this is better - the headline "Using CLI via Docker" is in a file in the LightGBM repo with h1 "Using LightGBM via Docker", what other CLI can we be talking about here considering the context? I've made the change nevertheless.

Reply:
The reason was to add the word "Version". I agree that "... of LightGBM" is unnecessary, you can remove it :-).